setup.py

```diff
@@ -38,9 +25,10 @@
     commit = p.stdout.readlines()[0].split()[1].decode("utf-8")
 except Exception:
     commit = ""
+
 f = open("pcmdi_metrics/version.py", "w")
-print("__version__ = '%s'" % Version, file=f)
-print("__git_tag_describe__ = '%s'" % descr, file=f)
+print("__version__ = '%s'" % release_version, file=f)
+print("__git_tag_describe__ = '%s'" % release_version, file=f)
```

Comment:
@tomvothecoder, I think it doesn't make sense to keep this line if you're eliminating the `git describe` call.  It would give the wrong idea.  That wasn't what I had recommended, actually.  I was just recommending that release_version be used below in the `setup()` call.